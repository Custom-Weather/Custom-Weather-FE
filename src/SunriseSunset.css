.sunrise-sunset {
  text-align: center;
}

.sunrise-text {
  display: inline-block;
  position: fixed;
  top: 46%;
  left: 43.5%;
  background-color: white;
  z-index: 1;
}

.sunset {
  display: inline-block;
  position: fixed;
  top: 46%;
  left: 53.5%;
  background-color: white;
  z-index: 1;
}
#sunrise-img {
  width: 10%;
  position: fixed;
  height: 10%;
  top: 40%;
  left: 40%;
  background-color: white;
  padding: 10px;
  padding-top: 0px;
  padding-bottom: 8px;
  border-top-left-radius: 8px;
  border-bottom-left-radius: 8px;
}

#sunset-img {
  width: 8%;
  height: 8%;
  left: 51%;
  top: 40%;
  position: fixed;
  background-color: white;
  padding: 10px;
  padding-bottom: 13px;
  padding-right: 25px;
  border-top-right-radius: 8px;
  border-bottom-right-radius: 8px;
}
