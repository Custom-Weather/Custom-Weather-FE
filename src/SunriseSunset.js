import React from 'react';
import './SunriseSunset.css';

const SunriseSunset = ({ sunrise, sunset }) => {
  console.log(sunrise)
  return (
    <div className='sunrise-sunset'>
      <h4> Sunrise and Sunset </h4>
      <p>Sunrise: {sunrise}</p>
      <p>Sunset: {sunset}</p>
    </div>
  )
}

export default SunriseSunset;
