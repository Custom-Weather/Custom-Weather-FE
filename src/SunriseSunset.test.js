import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';
import SunriseSunset from './SunriseSunset';

describe("SunriseSunset", () => {
  let mockRender;
  beforeEach(() => {
    mockRender=render(
      <SunriseSunset
      sunrise={"11:33 AM"}
      sunset={"2:44 AM"}
      />
    )
  })

  afterEach(() => {
    cleanup()
  })

  it("should render hourly forecasts", () => {
    const { getByText } = mockRender;
    const sunriseEl = getByText("11:33 AM")
    expect(sunriseEl).toBeInTheDocument();
    const sunsetEl = getByText("2:44 AM")
    expect(sunsetEl).toBeInTheDocument();
  })
})
