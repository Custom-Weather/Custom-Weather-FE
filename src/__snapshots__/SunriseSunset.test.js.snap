// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="sunrise-sunset"
  >
    <h4>
       Sunrise and Sunset 
    </h4>
    <p>
      Sunrise: 
    </p>
    <p>
      Sunset: 
    </p>
  </div>
</DocumentFragment>
`;
